monkey/infection_monkey/monkey.py

```diff
@@ -469,8 +472,13 @@ def log_arguments(self):
 
     @staticmethod
     def run_ransomware():
+        telemetry_messenger = LegacyTelemetryMessengerAdapter()
+        batching_telemetry_messenger = BatchingTelemetryMessenger(telemetry_messenger)
+
         try:
-            telemetry_messenger = TelemetryMessengerWrapper()
-            RansomewarePayload(WormConfiguration.ransomware, telemetry_messenger).run_payload()
+            RansomewarePayload(
+                config=WormConfiguration.ransomware,
+                telemetry_messenger=batching_telemetry_messenger,
+            ).run_payload()
```

Comment:
`config=WormConfiguration...` and `telemetry_messenger=batching_telemetry_messenger` both seem redundant to me. Without the parameter names, is there any question in your mind that if I'm passing in `batching_telemetry_messenger` as an argument, it's being assigned to the `telemetry_messenger` parameter?